.github/workflows/ci-test-pr.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
     - master
```

Comment:
I noticed this before, I think we should add `develop` to this list of `branches` too.

Reply:
I don't think we need to run tests on develop since they're required to pass on the PR before merging, if it gets to develop branch it should just go straight to deploy